semantics/language/common/expr/eval.k

```diff
@@ -41,19 +43,19 @@ module C-COMMON-EXPR-EVAL
      }*/
 
      // Lvalue conversion.
-     rule reval(nclv(Loc:SymLoc, T:Type)
+     rule reval(nclv(Loc::SymLoc, T::Type)
```

Comment:
I don't like this change. It may be faster, but it works against the type system. Next thing we know we have stuff other than ```SymLoc``` as the first argument because it type checks.